src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -2725,12 +2728,12 @@ class AccessibilityHelpWidget extends Widget implements ITerminalWidget {
 		this._domNode.setAttribute('role', 'dialog');
 		this._domNode.setAttribute('aria-hidden', 'true');
 		this._domNode.appendChild(this._contentDomNode);
-		this.onblur(this._contentDomNode.domNode, () => this.hide());
 		this._register(dom.addStandardDisposableListener(this._contentDomNode.domNode, 'keydown', (e) => {
-			if (e.keyCode === KeyCode.Escape) {
+			if (e.keyCode === KeyCode.Escape || e.keyCode === KeyCode.Tab) {
```

Comment:
Don't we want tab to focus the link inside the a11y help?

Reply:
it could, but i'm not sure that's expected - it's not the only point of interest within the document.

Reply:
Tab should move between interactive elements like links though, not doing that breaks common behavior